README.adoc

```diff
@@ -335,43 +352,37 @@ More details: https://mercedes-benz.github.io/sechub/latest/sechub-client.html#i
 
 The `licenseScan` module scans code or artifacts for license information.
 
-PDS-Solutions:
-
-* Scancode
-* Tern
-
-[[secretscan]]
-==== secretScan
-
-**Status: Productive**
-
-Scans code or artifacts for secrets (API tokens, certificates, passwords).
-
-PDS-Solutions:
-
-* Gitleaks
+PDS-Solutions: +
+- Scancode +
+- Tern
 
 === Architecture
 
 SecHub is designed to execute hundreds of scans. It can scale horizontally and vertically. It can run on bare-metal, virtual machines, kubernetes or in the cloud.
 
 The smallest useful setup is: a single SecHub server and a single product delegation server (PDS). Those two components are enough to start scanning.
 
-For a larger setup, the number of SecHub server instances can be increased. More PDS instances can be added. The only requirements to scale SecHub are: a PostgreSQL database and an object store or file share. SecHub and PDS instances use the PostgreSQL database to share information between instances. For example, the job queue is kept in PostgreSQL. In addition,an object store or file share is necessary, so that all SecHub or PDS instances can store/read files.
+For a larger setup, the number of SecHub server instances can be increased. More PDS instances can be added. +
+The only requirements to scale SecHub are: a PostgreSQL database and an object store or file share.
+SecHub and PDS instances use the PostgreSQL database to share information between instances.
+For example, the job queue is kept in PostgreSQL. In addition,an object store or file share is necessary, so that all SecHub or PDS instances can store/read files.
 
 Regardless of the backend complexity, whether one SecHub server or many are used the <<user-perspective, workflow for the user>> stays the same.
 
 For more details about the architecture have a look at the architecture documentation: https://mercedes-benz.github.io/sechub/latest/sechub-architecture.html.
 
 === Operations Perspective
 
-The operations team has to do most of the work. The operations team of SecHub needs to configure the scan tools as well as manage users and projects. This is an intentional design choice. The idea is, that users like security experts, operations engineers and developers can just scan and therefore focus on their work, rather than take care of SecHub and security scanners.
+The operations team has to do most of the work.
+The operations team of SecHub needs to configure the scan tools as well as manage users and projects.
+This is an intentional design choice.
+The idea is, that users like security experts, operations engineers and developers can just scan and therefore focus on their work, rather than take care of SecHub and security scanners.
```

Comment:
Maybe we should cut this part out. Users might not split their teams in DEV and OPS and instead have the same people run and use SecHub.

Reply:
rewrote the section